### PR TITLE
Kepori Clothing expansion

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request
Hey, this expands some clothing Kepori can wear.

From this humble PR Kepori gain uniquely sprited shoes, oversuits, and gloves for Combat boots, Jackboots, the basic sneakers, workboots, and explorer boots, Inteq Battlecoat and inteq winter coats, nitriles and combat gloves.

Adds some copy pasta paths and relating code for anyone else wanting to add clothing for them in the future.

Spin around of the battlecoat, combat boots, and winter coat for Keppies.
Kep_spin

*Also* adds sprites for Keppie socks but I couldn't find the option to equip them in the test server so, I'm not entirely sure where that is.

## Why It's Good For The Game

Birdies!

## Changelog

:cl: Botanics, Ebin-halcyon
imageadd: Kepori have had some specific items uniquely resprited for them, this includes, basic shoes, combat boots, jackboots, underwear and more. Along with some inteq specific gear.
/:cl:

### CODE REVIEW

**Review by Imaginos16 — APPROVED** [review]
I want to see at least one testmerge to make sure nothing fucks up before I hit the merge button, but I love this already.

**Review by thgvr — APPROVED** [review]
Add underwear stuff to changelog, implementation looks fine as well

**Review by MarkSuckerberg — APPROVED** [review]
note to self make a better system for kepi and vox in the future

but that aside WOOOOO YEAHHHHH HELL YEAH

**Pull request merged**
